src/pdn/test/four_sides/pdn.cfg

```diff
@@ -0,0 +1,36 @@
+# Floorplan information - core boundary coordinates, std. cell row height,
+# minimum track pitch as defined in LEF
+
+set ::halo 4
+
+# POWER or GROUND #Std. cell rails starting with power or ground rails at the bottom of the core area
+set ::rails_start_with "POWER" ;
+
+# POWER or GROUND #Upper metal stripes starting with power or ground rails at the left/bottom of the core area
+set ::stripes_start_with "POWER" ;
+
+# Power nets
+set ::power_nets "VDD"
+set ::ground_nets "VSS"
+
+##===> Power grid strategy
+# Ensure pitches and offsets will make the stripes fall on track
+pdngen::specify_grid stdcell {
+    name grid
+    core_ring {
+        metal1 {width 3  spacing  3.0 core_offset 5}
+        metal2 {width 3  spacing  3.0 core_offset 5}
+        metal3 {width 3  spacing  3.0 core_offset 5}
+        metal4 {width 3  spacing  3.0 core_offset 5}
+    }
+    gnd_pads {
+        VSS {GPAD_LONG GPAD_SHORT}
+    }
+    pwr_pads {
+        VDD {PPAD_LONG PPAD_SHORT}
+    }
+    connect {
+        {metal1 metal2} {metal2 metal3} {metal3 metal4}
+    }
+}
```

Comment:
Its a strange design that has no rails or straps. The straps do interact with the connections to the core rings, so whouls probably be included.